### PR TITLE
HS-876 - Process to pull base img put in github prvt registry

### PR DESCRIPTION
**Pull request by jaymberry**
## Description
Process to pull base img put in github private registry.

## Jira link(s)
- https://opennms.atlassian.net/browse/HS-876

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: .github/base-image-tag-ref.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "image_tag_java": "eclipse-temurin:17-jre",
+  "test": "1",
```
Comment (geraldhumphries) [inline .github/base-image-tag-ref.json:3]: Is "test" used for anything?

Comment (jaymberry) [reply .github/base-image-tag-ref.json:3]: Yes, I use it for triggering ci-cd tests. I have these spread out through the repo, we can get rid of them at some point, but come in handy while still in early stage of development.
 
I will make a ticket to remvoe these by end of 2nd Quarter.

**Review by geraldhumphries — APPROVED** [review]
Approved. There is just a small amount of leftover testing data that should be removed but that should not block the merge.

File: .github/workflows/base-image-update.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Pull Image & Push
+        run: |
+
+          IMAGE=$(cat .github/base-image-tag-ref.json | jq .image_tag_java | sed 's/\"//g')
+
+          docker pull $IMAGE
```
Comment (artnaseef) [inline .github/workflows/base-image-update.yml:32]: I recommend quoting `"$IMAGE"` here - if spaces end up in the variable somehow, the quotes will make the errors more obvious and make sure the results are "correct".

**Review by artnaseef — APPROVED** [review]

File: .github/workflows/base-image-update.yml (hunk 2 above)
Comment (jaymberry) [reply .github/workflows/base-image-update.yml:32]: ok, will implement. Thank you

**Pull request merged**